src/lib/app.ts

```diff
@@ -177,6 +178,8 @@ export default async function getApp(
         rbacMiddleware(config, stores, services.accessService),
     );
 
+    app.use(baseUriPath, originMiddleware(config));
```

Comment:
Placement of the middleware is important. I think this works nicely (after auth and before most other things), but we may need to move it.

Reply:
Should we use 
```suggestion
    app.use("${baseUriPath}/admin", originMiddleware(config));
```?

Reply:
Good question, it depends on which routes we want to cover. This covers everything, but maybe we want to be more selective.

Reply:
I'm with Gaston on this one, I don't see a need to look at anything but what we want, and we know what we want lives on the admin API

Reply:
My take is to just cover /api/admin because I looked at our access logs and it's difficult to actually get any information out of them. We can query 1minute in loki but anything else becomes impossible. /admin should have less frequency and more value for our task

Reply:
Sounds good, addressed in https://github.com/Unleash/unleash/pull/7695/commits/d46d14f7a63a14d4151593125606cede72d849d2